### ISSUE TITLE
Request to adjust the release plan

### ISSUE THREAD

**Issue by FeynmanZhou** [oras-project/oras#1038]
Hi ORAS maintainers, I would like to request to adjust the ORAS CLI release plans as follows and clarify the release management process in the community.

### Cut a patch release for ORAS CLI v1.0.0

ORAS CLI v1.0.0 was released on Mar 20. As this is the first stable release, we may need to consider providing relatively "long-term" support for this release until v1.3.0 or v2 is out. I also add more clarity to the release support strategy in the ORAS release management guideline, see [this PR](https://github.com/oras-project/community/pull/53/files). 

ORAS CLI v1.1.0-RC.1 was released on July 13 and the [release-1.1 branch](https://github.com/oras-project/oras/tree/release-1.1) is in active maintenance over the past four months. Introducing necessary bug fixes from [release-1.1 branch](https://github.com/oras-project/oras/tree/release-1.1) to the [release-1.0 branch](https://github.com/oras-project/oras/tree/release-1.0) then cut a patch release (v1.0.1) could be an appropriate option to maintain ORAS v1.0.0. 

I would recommend ORAS CLI maintainers @oras-project/oras-cli merge all commits from [release-1.1 branch](https://github.com/oras-project/oras/tree/release-1.1) to [release-1.0 branch](https://github.com/oras-project/oras/tree/release-1.0) except for [this new feature](https://github.com/oras-project/oras/commit/ba60b995d7ba1f8cf2f1a8788b25678c60af8225). The detailed commit comparison is listed [here](https://github.com/oras-project/oras/compare/release-1.0...release-1.1). Afterward, we can cut a patch release ORAS v1.0.1 from the release-1.0 branch.

### Plan a new milestone ORAS CLI v1.1.0-RC.2 to support OCI Image-spec v1.1.0-rc.4

[ORAS CLI v1.1.0-rc.1](https://github.com/oras-project/oras/releases/tag/v1.1.0-rc.1) as a Minor release but there is only one new feature as a small optimization introduced in this release. It might not be attractive and valuable enough for existing users to upgrade. Considering [OCI Image-spec v1.1.0-rc.4](https://github.com/opencontainers/image-spec/releases/tag/v1.1.0-rc4) was released on June 23, ORAS as a client tooling implementation to comply with the latest release of OCI Spec, so we may need to consider prioritizing the OCI Image-spec v1.1.0-rc.4 support in the release plan. It will enable users to experience the latest functionalities of OCI Image Spec and enhance ORAS CLI v1.1.0.

I would recommend ORAS CLI maintainers @oras-project/oras-cli plan and create a v1.1.0-RC.2 milestone which focuses on [OCI Image-spec v1.1.0-rc.4](https://github.com/opencontainers/image-spec/releases/tag/v1.1.0-rc4) support as described in #1009. This also relies on [ORAS-go v2.3.0](https://github.com/oras-project/oras-go/milestone/14). We can discuss and confirm the ETA of ORAS v1.1.0-RC.2 in the next community meeting. The ETA of ORAS v1.1.0 could be planned two weeks later than v1.1.0-RC.2 for testing purposes.

**Comment by yizha1** [comment]
> Introducing necessary bug fixes from [release-1.1 branch](https://github.com/oras-project/oras/tree/release-1.1) to the [release-1.0 branch](https://github.com/oras-project/oras/tree/release-1.0)

Should the bug fixes be merged from `main` to `release-1.0` branch?

**Comment by shizhMSFT** [comment]
For `dev` perspective, the steps to release `v1.0.1` is

1. Merge `main` into `release-1.0`
2. On `release-1.0`, revert #957 
3. Create a vote `v1.0.1` PR targeting the `release-1.0` branch

**Comment by FeynmanZhou** [comment]
Good catch. Merge `main` into `release-1.0` should be the right option in this case if we agree on release v1.0.1. There are more bug fixes in the `main` branch when comparing with `release-1.1` branch.

**Comment by TerryHowe** [comment]
SGTM

**Comment by qweeah** [comment]
SGTM

**Comment by sajayantony** [comment]
LGTM.

**Comment by FeynmanZhou** [comment]
Close this issue since we have released ORAS v1.1.0
